src/devices/video/ppu2c0x_vt.h

```diff
@@ -0,0 +1,78 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+/******************************************************************************
+
+    VT video emulation
+	
+	The VT video is based on the ppu2c0x but with enhanced capabilities such
+	as 16 colour sprites.
+
+******************************************************************************/
+
+#include "emu.h"
```

Comment:
Please don't `#include "emu.h"` from headers - it messes with stuff because of the way we use it as a precompiled prefix header.